examples/linear_extension/linear_extension.jl

```diff
@@ -14,30 +14,30 @@ function run_linear_extension_example()
     # -------------------------------
     ## scalar parameters
     E = 2.0             # Young modulus in Pa
-    ν = 0.4             # Poisson's ratio
+    nu = 0.4             # Poisson's ratio
     p = 3               # Tension load in Pa
     tension(t) = p * t  # Tension load function
-    Lᵢ = 2.0            # Dimension in x of the box in m
-    Lⱼ = 1.0            # Dimension in y of the box in m
-    Lₖ = 1.0            # Dimension in z of the box in m
+    Lx = 2.0            # Dimension in x of the box in m
+    Ly = 1.0            # Dimension in y of the box in m
+    Lz = 1.0            # Dimension in z of the box in m
     RTOL = 1e-4         # Relative tolerance for tests
     NSTEPS = 9          # Number of steps for the test
     ms = 0.5            # Refinement factor
     # -------------------------------
     # Materials
     # -------------------------------
     mat_label = "mat"
-    mat = IsotropicLinearElastic(E, ν, mat_label)
+    mat = IsotropicLinearElastic(E, nu, mat_label)
```

Comment:
The issue with `\nu` is that we're using this symbol in constructors, so when they are used as kwarg, the change wont apply -- would be better to fix this on the user end imho. 

I had a quick look and the reason the unicode is printed as boxes is due to missing fonts.

Reply:
For example see constructors around https://github.com/ONSAS/ONSAS.jl/blob/main/src/Materials/SvkMaterial.jl#L49

Reply:
this is not an issue in  `IsotropicLinearElastic` but indeed we have to discuss this for changes in other examples